CHANGELOG_PENDING.md

```diff
@@ -38,6 +39,9 @@
 - [rpc] \#3362 `/dial_seeds` & `/dial_peers` return errors if addresses are incorrect (except when IP lookup fails)
 - [node] \#3362 returns an error if `persistent_peers` list is invalid (except when IP lookup fails)
 - [p2p] \#3531 Terminate session on nonce wrapping (@climber73)
+- [libs/db] \#3611 Conditional compilation
+  * Use `cleveldb` tag instead of `gcc` to compile Tendermint with CLevelDB or
```

Comment:
We have to be careful that by moving from `gcc` to some custom build flag we did not change the default behaviour. When `CGO_ENABLED=1` the `cgo` (and the `gcc`?) flag would have been implicitly be set. Not true for `cleveldb`. 
Should the changelog state which backend will be used by default?

Reply:
> Should the changelog state which backend will be used by default?

mm, I thought everybody knows it is goleveldb and we did not change it here, so why state it?

Reply:
> When CGO_ENABLED=1 the cgo (and the gcc?) flag would have been implicitly be set. Not true for cleveldb.

good point 👍 but I think we still want to make it implicit - i.e. if you want to compile with cleveldb, please provide cleveldb build constraint, etc.

Reply:
*explicit